app/build.gradle

```diff
@@ -72,8 +72,7 @@ android {
   productFlavors {
     cobrand {
       dimension "product"
-      applicationId "com.aptoide.partners"
-      applicationIdSuffix project.COBRAND_APPLICATION_ID_SUFFIX.toString()
+      applicationId "com.aptoide." + project.COBRAND_APPLICATION_ID_SUFFIX.toString()
```

Comment:
package name = com.aptoide.partners.[store_name]

missing partners

Reply:
I was under the impression the partners was to be dropped. The "partners" was the problem with the mismatching WS auth and unability to login. 
I will look into it.

Thanks!